openwisp_notifications/templates/openwisp_notifications/unsubscribe.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'account/base_entrance.html' %}
+{% load i18n %}
+{% load static %}
+
+{% block head_title %}
+  <title>{% trans 'Manage Subscription Preferences' %}</title>
+{% endblock %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'openwisp-notifications/css/unsubscribe.css' %}">
+{% endblock %}
+
+{% block menu-bar %}
+{% endblock %}
+
+{% block content %}
+  <div class="unsubscribe-container">
+    <img
+      src="{% static 'ui/openwisp/images/openwisp-logo-black.svg' %}"
+      alt="{% trans 'OpenWISP Logo' %}"
+      class="logo"
+    />
+    <div class="unsubscribe-content">
+      <div class="icon email-icon"></div>
+      <h1>{% trans 'Manage Notification Preferences' %}</h1>
+      {% if valid %}
+        <p id="status-message">
+          {% if is_subscribed %}
+            {% trans 'You are currently subscribed to notifications.' %}
+          {% else %}
+            {% trans 'You are currently unsubscribed from notifications.' %}
+          {% endif %}
+        </p>
```

Comment:
We are duplicating these text messages in JS. Maybe, we can create separate paragraph elements for each condition and hide one using CSS based on `is_subscribed`